clients/unshaded/src/main/java/tachyon/client/TachyonFSTestUtils.java

```diff
@@ -77,11 +76,25 @@ public static TachyonFile createByteFile(TachyonFileSystem tfs, String fileName,
   public static TachyonFile createByteFile(TachyonFileSystem tfs, TachyonURI fileURI,
       TachyonStorageType tachyonStorageType, UnderStorageType underStorageType, int len)
       throws IOException {
+    OutStreamOptions options = new OutStreamOptions.Builder(ClientContext.getConf())
+        .setTachyonStorageType(tachyonStorageType).setUnderStorageType(underStorageType).build();
+    return createByteFile(tfs, fileURI, options, len);
+  }
+
+
+  /**
+   * Creates a simple file with <code>len</code> bytes.
+   *
+   * @param tfs a TachyonFileSystem handler
+   * @param fileURI URI of the file
+   * @param options client options to create the file with
+   * @param len file size
+   * @return the TachyonFile of the created file
+   * @throws IOException if <code>path</code> is invalid (e.g., illegal URI)
+   */
+  public static TachyonFile createByteFile(TachyonFileSystem tfs, TachyonURI fileURI,
+      OutStreamOptions options, int len) throws IOException {
```

Comment:
there were two implementations of the same method, one takes a string `fileName` and the other takes a TachyonURI. The difference between the two methods were non significant. I don't see why we want to keep them separate. Isn't it more maintainable to have only one impl and have everyone else calling it?